jenkins/common.groovy

```diff
@@ -253,7 +253,7 @@ def buildAll(String ref, dockerContainer, Boolean contentOnlyBuild) {
 
 def prearchive(dockerContainer, envName) {
   dockerContainer.inside(DOCKER_ARGS) {
-    sh "cd /application && node script/prearchive.js --buildtype=${envName}"
+    sh "cd /application && node --max-old-space-size=4096 script/prearchive.js --buildtype=${envName}"
```

Comment:
This step was failing in Jenkins, as described [here](https://github.com/department-of-veterans-affairs/vets-website/pull/12704#issuecomment-630447026). Bumping the RAM allowance the same way we do for our unit tests, makes things happy.